api/lib/application/certification-centers/certification-center-controller.js

```diff
@@ -0,0 +1,45 @@
+const usecases = require('../../domain/usecases');
+const certificationCenterSerializer = require('../../infrastructure/serializers/jsonapi/certification-center-serializer');
+const controllerReplies = require('../../infrastructure/controller-replies');
+const {
+  NotFoundError: InfrastructureNotFoundError,
+} = require('../../infrastructure/errors');
+
+const {
+  NotFoundError,
+} = require('../../domain/errors');
+
+module.exports = {
+
+  save(request, h) {
+    const certificationCenter = certificationCenterSerializer.deserialize(request.payload);
+    return usecases.saveCertificationCenter({ certificationCenter })
+      .then((certificationCenter) => certificationCenterSerializer.serialize(certificationCenter))
+      .catch((error) => {
```

Comment:
Pareil ici : 
.then(controllerReplies(h).error)

Reply:
oui

Reply:
non

Reply:
ok je vois, on est pas à l'abri d'un this dans une des méthodes de happy qui se balade c'est ça?